src/cli.py

```diff
@@ -8,19 +8,19 @@
 from logic import generate_graph, print_version
 
 
-# @click.group()
+@click.group()
+@click.option('--version', is_flag=True, callback=print_version,
+              expose_value=False, is_eager=True)
+def cli():
+    pass
+
 @click.command()
 @click.option('--directory', '-d', help='Directory to analyze', required=True)
-def cli(directory: str):
+def dependencies_graph(directory: str):
```

Comment:
given that our CLI will have only one command, I would prefer to keep it as it previously was.

Reply:
makes sense